Bottom: 81b8bdaa4843e78c381016834c12108eaebcfefc
Top:    90d0866a7aee2e6efd9dee61907b938b12c1c2b6
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-04 17:21:00 -0500

Refresh of buildbot-master

---
diff --git a/ansible-playbook/group_vars/all/buildbot.yml b/ansible-playbook/group_vars/all/buildbot.yml
index 29d7c9f8..c918df8f 100644
--- a/ansible-playbook/group_vars/all/buildbot.yml
+++ b/ansible-playbook/group_vars/all/buildbot.yml
@@ -1,5 +1,5 @@
 ---
-buildbot_version: 1.1.1
+buildbot_version: 1.1.2
 buildbot_worker_port: 9989
 buildbot_worker_ntel: worker-ntel
 buildbot_worker_ptxdist: worker-ptxdist
